OpenIPSL/Electrical/Sources/SourceBehindImpedance/VoltageSources/VSourceIO.mo

```diff
@@ -0,0 +1,117 @@
+within OpenIPSL.Electrical.Sources.SourceBehindImpedance.VoltageSources;
+model VSourceIO
+  "Generic voltage source behind an internal impedance with inputs that vary the internal source magnitude and angle from steady state"
+  extends BaseClasses.baseVoltageSource;
+  // Input ports and polar to rectangular conversion
+public
```

Comment:
public is not needed since there is no protected

Reply:
Fixed.